nerdlets/util/async.js

```diff
@@ -232,19 +232,19 @@ async function processBatchOfCardinalities(
 ) {
   let failedBatch = false;
   try {
-    const { data, errors } = await calculateCardinalityForRuleBatch(
+    const { data, error } = await calculateCardinalityForRuleBatch(
       batchOfRules,
       accountId
     );
-    if (errors) {
-      const errorStr = JSON.stringify(errors);
+    if (error) {
+      const errorStr = JSON.stringify(error.graphQLErrors);
```

Comment:
@aswanson-nr not sure if this was necessary - i can change it otherwise

Reply:
specifically `error.graphQLErrors` instead of just `error`

Reply:
Looks fine to me! Especially since it's being parsed for a string later